Basic/DistributedAuthoritySocialHub/Assets/Editor/Tutorials/Tutorial0_SetupUGS/2-CheckServices.asset

```diff
@@ -0,0 +1,211 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
+--- !u!114 &11400000
+MonoBehaviour:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  m_GameObject: {fileID: 0}
+  m_Enabled: 1
+  m_EditorHideFlags: 0
+  m_Script: {fileID: 11500000, guid: ff771ccdf4150419d9ff4d342b069aae, type: 3}
+  m_Name: 2-CheckServices
+  m_EditorClassIdentifier: 
+  Title:
+    m_Untranslated: Services General Settings
+  m_Paragraphs:
+    m_Items:
+    - m_Type: 6
+      Title:
+        m_Untranslated: 
+      Text:
+        m_Untranslated: 
+      m_CodeSample: 
+      m_PostInstructionImage: {fileID: 0}
+      m_Tutorial: {fileID: 0}
+      m_Image: {fileID: 0}
+      m_VideoUrl: 
+      m_Video: {fileID: 0}
+      m_CriteriaCompletion: 0
+      m_Criteria:
+        m_Items: []
+      m_MaskingSettings:
+        m_MaskingEnabled: 1
+        m_UnmaskedViews:
+        - m_SelectorType: 1
+          m_ViewType:
+            m_TypeName: 
+          m_FocusEditorWindow: 1
+          m_EditorWindowType:
+            m_TypeName: UnityEditor.ProjectSettingsWindow, UnityEditor.CoreModule,
+              Version=0.0.0.0, Culture=neutral, PublicKeyToken=null
+          m_AlternateEditorWindowTypes:
+            m_Items: []
+          m_MaskType: 0
+          m_MaskSizeModifier: 0
+          m_UnmaskedControls: []
+      m_Summary: 
+      m_Description: 
+      m_InstructionBoxTitle: 
+      m_InstructionText: 
+      m_TutorialButtonText: 
+    - m_Type: 0
+      Title:
+        m_Untranslated: 
+      Text:
+        m_Untranslated: "To configure your project to use Unity Cloud Services:\n\n1.
+          Go to <b>Edit</b> and select the <b>Project Settings</b> window. \n\n2.
```

Comment:
Step 1 and 2 are not needed, as the Project Settings window opens automatically when you enter this tutorial and it already points to the Services" tab.

Step 3 should highlight the Organization popup if possible (IET has a feature for that)

Bold is really over-used here. As per the [design guidelines](https://docs.google.com/presentation/d/1aEJkrIhvJ4hf-IXOghLvl0Y3_u7-hPd0aHlqa3RKddc/edit#slide=id.p), you should use it for important and specific terms, not to highlight actions in the middle of narrative paragraphs. (I.E: "Use an existing project ID" should not go entirely in bold).

Slide 15 of the guidelines says to avoid italic, and half of the NOTE paragraph is in italic.

If this approach was reused across tutorials of this project, I won't comment every single instance of it, but keep i mind that the feedback applies to all of them.

Reply:
Also, clicking this link has no impact

![image](https://github.com/user-attachments/assets/b4f1ebc2-b445-4378-8ec8-5bd6a3e69d5a)

Reply:
Same point as above on the styling.
Though the broken link I've fixed!